scripts/common/rook.sh

```diff
@@ -293,25 +295,27 @@ function maybe_cleanup_rook() {
 
         # If upgrade from Rook to OpenEBS without Minio we cannot remove Rook because
         # we do not know if the solution uses or not ObjectStore and if someone data will not be lost
-        if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ -z "$MINIO_VERSION" ]; then
+        if [ -n "$DID_MIGRATE_ROOK_PVCS" ] && [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ -z "$MINIO_VERSION" ]; then
```

Comment:
this change is not necessary
```suggestion
        if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ -z "$MINIO_VERSION" ]; then
```